benchmarks/hyperspecialization/list-generate/baseline.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+import time
+import generate_data
+
+
+def simple_func(simple_list):
+    my_dict = {}
+    for element in simple_list:
+        if element in my_dict:
+            my_dict[element] += 1
+        else:
+            my_dict[element] = 0
+
+    return my_dict
+
+def run_simple_func(list_of_lists):
+
+    tstart = time.time()
```

Comment:
let's maybe try a version of this where we use 
1.) [...] list comprehension
2.) map(...) and then a final list(...) call on the result of map.

==> Interested in how they perform relative to each other.